aaelf64-morello/aaelf64-morello.rst

```diff
@@ -0,0 +1,825 @@
+..
+   Copyright (c) 2011-2020, Arm Limited and its affiliates.  All rights reserved.
```

Comment:
The copyright range varies per document. As this document was created this year, this should just read `2020`, instead of `2011-2020`.